test/e2e/workflow/validation_test.go

```diff
@@ -13,6 +13,9 @@ import (
 
 var _ = Describe("NetworkAddonsConfig", func() {
 	gvk := GetCnaoV1GroupVersionKind()
+	AfterEach(func() {
```

Comment:
This should be already done here https://github.com/kubevirt/cluster-network-addons-operator/blob/master/test/e2e/workflow/main_test.go#L59. If it is not, it is a bug in the suite cleanup, no?

Reply:
because the inner gvk wins ?
https://github.com/kubevirt/cluster-network-addons-operator/pull/672/files#diff-fddb4e4e7a7cebcb6649159ab67dfa1c36da1d4b193f1d2e19e93638c7a7b11eR15

Reply:
or perhaps it's not a good fix at all
I think you're right Petr, will further investigate it. closing the PR in the meantime